css/style.css

```diff
@@ -309,3 +338,105 @@ input {
   color: #ebebff;
   margin-bottom: 40px;
 }
+
+@media (min-width: 768px) {
+  .findMe {
+    width: 768px;
+    justify-content: space-between;
+  }
+
+  .work {
+    margin: 1px;
+    padding-left: 10px;
+    padding-right: 10px;
+  }
+
+  .habilities {
+    width: 600px;
+    margin-left: auto;
+    margin-right: auto;
+    grid-template-columns: auto auto auto;
+    grid-template-rows: auto auto;
+    place-items: center;
+  }
+
+  .description {
+    width: 600px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .title {
+    background-image: url("../pictures/twocirclesheader.png");
+    background-position: center;
+  }
```

Comment:
- [ ] The background image disappears when the width is around 900px:

![image](https://user-images.githubusercontent.com/20567503/177862264-4eaf26c0-a40c-4090-9624-43f785f90c83.png)

To fix that kindly add this line `background-size: contain;` 😉